IFC4x3/Templates/Product Shape/Product Geometric Representation/Alignment Geometry/Alignment Geometry Gradient/DocTemplateDefinition.xml

```diff
@@ -37,6 +37,51 @@
 										</DocModelRuleAttribute>
 									</Rules>
 								</DocModelRuleEntity>
+								<DocModelRuleEntity Name="IfcShapeRepresentation">
+									<Rules>
+										<DocModelRuleAttribute Name="ContextOfItems">
+											<Rules>
+												<DocModelRuleEntity Name="IfcGeometricRepresentationContext" />
+											</Rules>
+										</DocModelRuleAttribute>
+										<DocModelRuleAttribute Name="RepresentationIdentifier">
+											<Rules>
+												<DocModelRuleEntity xsi:nil="true" href="IfcLabel_1UYMuwZ1566P968r8_v2Pz" />
+											</Rules>
+										</DocModelRuleAttribute>
+										<DocModelRuleAttribute Name="RepresentationType">
+											<Rules>
+												<DocModelRuleEntity xsi:nil="true" href="IfcLabel_0hxnaRHwbFuxyZ3b2iuiyK" />
+											</Rules>
+										</DocModelRuleAttribute>
```

Comment:
Literals for comparison missing.

Reply:
Think this is also resolved with the latest commit. Please check and resolve the comment.

Reply:
This doesn't seem to be resolved. Or are there no `RepresentationIdentifier` and `RepresentationType` constraints imposed by this concept template?